packages/zowe-explorer/src/shared/context.ts

```diff
@@ -386,10 +386,14 @@ export function asFavorite(node: TreeItem): string {
 }
 
 export function withProfile(node: IZoweTreeNode): string {
-    if (node?.contextValue?.includes(".profile=")) return node?.contextValue;
-    return node?.getParent()?.contextValue?.includes(".profile=")
-        ? node?.contextValue + ".profile=" + node?.getParent()?.contextValue?.split(".profile=")[1] ?? node?.contextValue
-        : node?.contextValue;
+    if (!node) return;
```

Comment:
Looks like the linter wants braces for these one-line `if` statements 😅 (same w/ line 391)